RMSwiftUI/RMSwiftUI/ContentView.swift

```diff
@@ -7,15 +7,158 @@
 
 import SwiftUI
 
+/// Экран с примерами работы Alert и ActionSheet
 struct ContentView: View {
+    @State private var isPresentedFirstAlert = false
+    @State private var isPresentedSecondAlert = false
+    @State private var isPresentedFirstActionSheet = false
+    @State private var isPresentedSecondActionSheet = false
```

Comment:
```suggestion
    @State private var isFirstAlertShown = false
    @State private var isSecondAlertShown = false
    @State private var isFirstActionSheetShown = false
    @State private var isSecondActionSheetShown = false
```
 если флаг что-то показывает, то на конце Shown